test/test-functions

```diff
@@ -442,8 +455,14 @@ install_config_files() {
     # set the hostname
     echo systemd-testsuite > $initdir/etc/hostname
     # fstab
+    if [[ "$LOOKS_LIKE_SUSE" ]]; then
+       ROOTMOUNT="/dev/sda1           /       ${FSTYPE}    rw 0 1"
+    else
+       ROOTMOUNT="LABEL=systemd           /       ${FSTYPE}    rw 0 1"
```

Comment:
> The background ist that we intend to use the testsuite not only in the build environment but also in our QA.

The more tests, the better 👍 

I think it would also be great to run SUSE tests when a PR is opened for `systemd` on GitHub.